src/Item.cpp

```diff
@@ -4,26 +4,42 @@
 namespace tt
 {
 
-    Item::Item( sf::Texture texture,
+    Item::Item( std::string id,
```

Comment:
It is generally not advisable to pass non-trivial objects by value, as it creates a copy, and in most cases you don't need a copy. 

Instead use a `const-ref` like so:

```
Item::Item(const std::string& id, ...
```

This avoids an extra copy.